### PR TITLE
[ADD] timesheet: doc about creating timesheets upon time off val…

### PR DESCRIPTION
**Pull request by kbr-odoo**
…idation

### CODE REVIEW

File: timesheets/overview/time_off.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+needed type, and decide if you would like the requests to be validated or not.
+
+.. image:: media/time_off_types.png
+   :align: center
+   :alt: View of a time off types form emphasizing the time off requests and timesheets section in
+         Odoo Time Off
+
+| Now, once the employee has requested his time off, and the request has been validated (or not,
+  depending on the setting chosen), the time is automatically allocated on *Timesheets*, under the
+  respective project and task.
```
Comment (Lau-Odoo) [inline timesheets/overview/time_off.rst:26]:
```suggestion
| Now, once the employee has requested his time off and the request has been validated (or not,
  depending on the setting chosen), the time is automatically allocated on *Timesheets*, under the
  respective project and task.
```

File: timesheets/overview/time_off.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+| On the example below, the user requested *Paid Time off* from July 13th to 15th.
+
+.. image:: media/time_off_request.png
+   :align: center
+   :alt: View of the time off request form in Odoo Time Off
+
+Considering that validation is not required, the requested time off is automatically displayed in
+*Timesheets*. If validation was necessary, the time is automatically allocated after the responsible
+person for validating does it so.
```
Comment (Lau-Odoo) [inline timesheets/overview/time_off.rst:35]:
```suggestion
*Timesheets*. If validation is necessary, the time is automatically allocated after the responsible
person for validating does it so.
```

File: timesheets/overview/time_off.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+.. image:: media/timesheets.png
+   :align: center
+   :alt: Video of timesheets emphasizing the requested time off from the employee in Odoo Timesheets
+
+Click on the magnifying glass, hovering over the concerned cell, to access all the aggregated data
+on that cel (day), and see details regarding the project/task.
```
Comment (Lau-Odoo) [inline timesheets/overview/time_off.rst:42]:
```suggestion
on that cell (day), and see details regarding the project/task.
```

**Comment by kbr-odoo** [comment]
@Lau-Odoo Thank you :)

**Pull request merged**